packages/shared/src/styles/components/menu.css

```diff
@@ -2,7 +2,7 @@
   position: fixed;
   opacity: 0;
   user-select: none;
-  z-index: 100;
+  z-index: 1000;
```

Comment:
Shouldnt we move these to named?

Reply:
Also above 10 and 100 both resolve to 1000 now?

Reply:
I am not sure what else I need to change to move them, so just updated for now. 

> Also above 10 and 100 both resolve to 1000 now?

not sure what you mean by this?

Reply:
Got it on the changes.
As for the other point, never mind my phone version showed one under 10 and one 100

Reply:
Yeah, all are basically multiplied by 10, so 10 -> 100, 100 -> 1000.